src/app_service/service/transaction/service.nim

```diff
@@ -532,11 +537,11 @@ QtObject:
 
         let network = self.networkService.getNetworkByChainId(chainID)
         if not network.isNil and network.nativeCurrencySymbol == asset.symbol:
-          self.transferEth(fromAddr, toAddr, asset.symbol, toAsset.symbol, uuid, selectedRoutes, finalPassword, sendType)
+          self.transferEth(fromAddr, toAddr, asset.symbol, toAsset.symbol, uuid, selectedRoutes, finalPassword, sendType, slippagePercentage)
```

Comment:
we probably need to rework this and use a struct instead of an eternal list of params. The status-go part of this is getting reworked in a couple of weeks so I guess it can be done whenever we integrate that.